packages/ui/.eslintrc.json

```diff
@@ -0,0 +1,33 @@
+{
+    "extends": ["../../.eslintrc.json"],
+    "ignorePatterns": ["!**/*"],
+    "overrides": [
+        {
+            "files": ["*.ts", "*.tsx", "*.js", "*.jsx"],
+            "rules": {
+                "@typescript-eslint/naming-convention": "off",
+                "@typescript-eslint/no-unsafe-assignment": "off",
+                "@typescript-eslint/no-misused-promises": "off",
+                "@typescript-eslint/no-unsafe-member-access": "off",
+                "@typescript-eslint/restrict-template-expressions": "off",
+                "@typescript-eslint/require-await": "off",
+                "@typescript-eslint/no-unsafe-argument": "off"
+            }
+        },
+        {
+            "files": ["*.spec.ts"],
+            "rules": {
+                "@typescript-eslint/no-unsafe-assignment": "off",
+                "@typescript-eslint/consistent-type-assertions": "off",
+                "@typescript-eslint/no-explicit-any": "off",
+                "@typescript-eslint/no-unsafe-argument": "off",
+                "@typescript-eslint/await-thenable": "off",
+                "jest/no-conditional-expect": "off",
+                "@typescript-eslint/no-floating-promises": "off",
+                "jest/valid-expect": "off",
+                "@typescript-eslint/no-unnecessary-condition": "off",
+                "@typescript-eslint/no-unsafe-call": "off"
```

Comment:
LGTM.
This list appears to be lengthy. I just want to point out that we intend to keep it to an essential minimum.

Reply:
Cleaned this list
I take this list from one of other packages, and currently see that all packages have different rules inside, it is little bit confusing.
Maybe in feature we can add single source of rules for all of them?

Reply:
We use https://github.com/bigcommerce/eslint-config as the base settings for all packages.
However, since we applied it in the middle of the last year, a lot of old code could not meet these standards, we had set a lot of rules off [in the core](https://github.com/bigcommerce/checkout-js/blob/master/packages/core/.eslintrc.json) and kept the rest packages' off settings at a minimum.